corehq/apps/reports/standard/monitoring.py

```diff
@@ -969,8 +969,8 @@ def generate_chart(cls, data, width=950, height=300, timezone="UTC"):
             sizes.extend([1] * 24)
         chart.add_data(sizes)
 
-        chart.set_axis_labels('x', [''] + [(str(h)+'h') for h in range(24)] + [''])
-        chart.set_axis_labels('x', [' ', 'Time ({})'.format(timezone), ' '])
+        chart.set_axis_labels('x', [''] + [(str(h) + 'h') for h in range(24)] + [''])
```

Comment:
This line might also need to be translated, but not quite sure how - this will show up as "0h", "1h", "2h", and so on.  One option would be to just use the number here and add a text label for the whole axis that's `_("hours")`